src/types/Contact.ts

```diff
@@ -3,8 +3,9 @@ import type { UploadedImage } from '@prezly/uploads';
 import type { ContactDuplicateSuggestion } from './ContactDuplicateSuggestion';
 import type { Entity } from './Entity';
 
-export interface ContactRef extends Entity {
-    contact_type: 'person' | 'organisation';
+export interface ContactRef extends Entity<number> {
+    id: number;
```

Comment:
Is this needed, if `id` property is inherited from the `Entity` type? Same on line 20

Reply:
Well, it's not really necessary. But it also makes things more clear -- it's immediately obvious that there's `id` property. So I'm hesitant to remove it.

Reply:
I was about to say that in that case you might as well ditch the `Entity` interface, but I see you already did :)